### PR TITLE
Issue #SB-24156 fix: Removed the infinite-scroll class from home component

### PR DESCRIPTION
**Pull request by christyfernandes**

### CODE REVIEW

File: projects/discussion-ui/src/lib/services/config.service.ts [hunk 1]
```diff
@@ -70,4 +70,8 @@ export class ConfigService implements OnInit {
   public getRouterSlug() {
     return this._config.routerSlug ? this._config.routerSlug : '';
   }
+
+  public getHeaderOption() {
```
Comment (christyfernandes) [inline projects/discussion-ui/src/lib/services/config.service.ts:74]: Why this method is added?

File: projects/discussion-ui/src/lib/elements/related-discussion/related-discussion.component.ts [hunk 2]
```diff
@@ -39,8 +39,9 @@ export class RelatedDiscussionComponent implements OnInit, OnChanges {
     this.discussionService.fetchSingleCategoryDetails(cid).subscribe(
       (data: NSDiscussData.ICategoryData) => {
         this.relatedDiscussions = [];
+        // filter to get the not deleted topics
         _.filter(data.topics, (topic) => {
-          if (this.topicId != topic.tid) {
+          if (topic.deleted == 0 && this.topicId != topic.tid) {
```
Comment (christyfernandes) [inline projects/discussion-ui/src/lib/elements/related-discussion/related-discussion.component.ts:44]: Why this condition is added ?

File: projects/discussion-ui/src/lib/services/config.service.ts (hunk 1 above)
Comment (bandanasahu123) [reply projects/discussion-ui/src/lib/services/config.service.ts:74]: This method has been added to remove the back and cancel button from igot. And these are main configurable.

File: projects/discussion-ui/src/lib/elements/related-discussion/related-discussion.component.ts (hunk 2 above)
Comment (bandanasahu123) [reply projects/discussion-ui/src/lib/elements/related-discussion/related-discussion.component.ts:44]: This condition has been added to get only the discussions which are not deleted from nodebb

**Pull request closed without merging**